src/lib/y2storage/boot_requirements_strategies/legacy.rb

```diff
@@ -166,22 +178,36 @@ def errors_on_gpt
       def errors_on_msdos
         errors = []
 
-        if !valid_mbr_gap?
-          errors << mbr_gap_error if !plain_btrfs?
-        elsif boot_partition_needed? && missing_partition_for?(boot_volume)
-          errors << SetupError.new(missing_volume: boot_volume)
+        if !mbr_gap_for_grub?
+          errors <<= mbr_gap_error
+          errors <<= if boot_can_embed_grub?
+            bad_config_warning
+          else
+            bad_config_error
+          end
         end
```

Comment:
How are these errors presented, as different pop-ups or in the same dialog one below the other?

Reply:
Presented as a bullet-list, looks ok.